src/components/atoms/SfBadge/README.md

```diff
@@ -0,0 +1,33 @@
+# SfBadge
+
+## Props
+
+- `type` - defines badge type, defaul "info"
+
```

Comment:
I think we don't need this prop, it may be handled by CSS modifiers only, we're using this approach on other components.

Reply:
Ok. deleted the type prop